app/templates/components/ui-table/cell/events/view/tickets/discount-codes/cell-url.hbs

```diff
@@ -1 +1,16 @@
-<a href='{{this.record}}' target='_blank' rel='noopener'> {{this.record}} </a>
+<div class="ui action input" style="width: 100%; height: 36px">
```

Comment:
Why

Reply:
In the same way it is in the cell video URL.

Reply:
Use URLBox component